meinberlin/apps/plans/models.py

```diff
@@ -22,6 +33,30 @@ class Plan(UserGeneratedContentModel):
                     'Drag and drop the marker to change its place. '
                     'Alternatively you can use the search field to search '
                     'for an address.'))
+    point_label = models.CharField(
+        blank=True,
+        default='',
+        max_length=255,
+        verbose_name=_('Label of the location'),
+        help_text=_('This could be an address or the name of a landmark.'),
+    )
+    contact = models.TextField(max_length=255, verbose_name=_('Contact'))
+    cost = models.PositiveIntegerField(blank=True, null=True,
+                                       verbose_name=_('Cost'))
+    description = RichTextField(verbose_name=_('Description'), blank=True)
```

Comment:
i think richtextfields need some additional handlin in the save method:
see https://github.com/liqd/a4-meinberlin/blob/master/meinberlin/apps/ideas/models.py#L44

although i don't now why o.0

Reply:
i did create issues on that topic #1011